### PR TITLE
BP Passport scanning not opening search results or patient summary

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/1965/bp-passport-scanning-not-opening-search-results-or-patient-summary

### CODE REVIEW

File: app/src/test/java/org/simple/clinic/home/HomeScreenEffectHandlerTest.kt [hunk 1]
```diff
@@ -56,4 +60,19 @@ class HomeScreenEffectHandlerTest {
...
+    val shortCode = "123456"
+
+    // when
+    effectHandlerTestCase.dispatch(OpenShortCodeSearchScreen(shortCode))
+
+    // then
+    effectHandlerTestCase.assertNoOutgoingEvents()
+
+    verify(uiActions).openShortCodeSearchScreen(shortCode)
+    verifyNoMoreInteractions(uiActions)
```
Comment (vinaysshenoy) [inline app/src/test/java/org/simple/clinic/home/HomeScreenEffectHandlerTest.kt:76]: We should also be verifying no interactions in the other test, right?

File: app/src/test/java/org/simple/clinic/home/HomeScreenUpdateTest.kt [hunk 2]
```diff
@@ -31,4 +31,22 @@ class HomeScreenUpdateTest {
...
+        patient = Optional.empty(),
+        identifier = identifier
+    )
+
+    updateSpec
+        .given(defaultModel)
+        .whenEvent(event)
+        .then(UpdateSpec.assertThatNext(
+            NextMatchers.hasNoModel(),
+            NextMatchers.hasEffects(OpenPatientSearchScreen(identifier))
```
Comment (vinaysshenoy) [inline app/src/test/java/org/simple/clinic/home/HomeScreenUpdateTest.kt:75]: Can we statically import these?

**Review by vinaysshenoy — APPROVED** [review]
Overall PR looks good, left a couple of comments that we can fix later.

**Review by vinaysshenoy — APPROVED** [review]

File: app/src/test/java/org/simple/clinic/home/HomeScreenEffectHandlerTest.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/test/java/org/simple/clinic/home/HomeScreenEffectHandlerTest.kt:76]: Good point missed it for one test. Fixing it

**Pull request merged**